backend/app/processing/free_llm_formatter.py

```
@@ -0,0 +1,80 @@
+    else:
+        raise Exception(f"HuggingFace API error: {response.status_code} - {response.text}")
+
+def format_with_ollama(prompt: str) -> str:
+    """Format response using local Ollama instance"""
+    payload = {
+        "model": "llama2",
+        "prompt": prompt,
+        "stream": False
+    }
+    
+    response = requests.post(
```

Ruff (0.8.2)

Probable use of `requests` call without timeout

(S113)